docs/releases/slashing-release/interfaces/slashing-parameter-configuration.md

```diff
@@ -59,29 +64,53 @@ interface IAVSDirectory {
 
     /// EXTERNAL - STATE MODIFYING
     
-    function queueAllocation(
+    /**
+     * @notice Allocate slashable magnitude for an operator.
+     * For each strategy, takes an array of magnitude adjustments(increments)
+     * and allocates them to their respective operatorSets. 
+     * Nonslashable magnitude for the strategy will decrement by the sum of all 
+     * allocations and these allocation increases will take effect 21 days(in blocks) from now.
```

Comment:
in timestamp?

Reply:
Default is using blocks for OZ checkpoints but we could also use timestamp as the keys. Probably makes more sense to use timestamps.